docs/model_library/Hydraulics/index.rst

```diff
@@ -205,4 +216,141 @@ Allows pumping only if a pump exists in a pipeline.
         \textcolor{red}{C_{l,\tilde{l}}^{Pump}} = \textcolor{green}{\nu^{Pump}} \cdot \textcolor{red}{y_{l,\tilde{l},[t]}^{Pump}}
         + \textcolor{green}{\nu^{Electricity}} \cdot \textcolor{green}{\rho.g} \cdot \sum_{t \in T}\textcolor{red}{H_{l,\tilde{l},t}^{Pump}} \cdot \textcolor{red}{F_{l,\tilde{l},t}^{Piped}}
 
+    
+**Linearizing the Equations/constraints specific to the co_optimize method**
+
+    **Parameters**
+    :math:`\textcolor{green}{\Delta_I}` =                        Length of interval of flows for building piecewise linear model
+    :math:`M` =                       Large enough constant
+
+    **Binary Variables**
+
+    :math:`\textcolor{red}{z_{l,\tilde{l},t, i}},i\in \{0,1,2,...,\{0,1,2,..., \lceil 70000/\textcolor{green}{\Delta_I}\rceil - 1}` =     Intermediate binary variables to determine the section of the piecewise linear graph.
```

Comment:
There is a problem somewhere in this line - this is how it is rendering currently: 
![image](https://github.com/project-pareto/project-pareto/assets/26798258/0fbf4710-7450-4a4b-ac5e-98b1c56e3c84)

Reply:
Thanks for the comment! I have removed repeated openings of curly braces.

Reply:
@Sourabhhkt it's still not rendering properly on the latest build.